test/acceptance/fixtures/static-server.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: static-server
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: static-server
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: static-server
+  template:
+    metadata:
+      name: static-server
+      labels:
+        app: static-server
+      annotations:
+        "consul.hashicorp.com/connect-inject": "true"
+    spec:
+      containers:
+        # This name will be the service name in Consul.
+        - name: static-server
+          image: ishustava/http-echo:latest
```

Comment:
did you build your own that actually shuts down on SIGTERM?

Reply:
I don't remember why I created my own image 😃 ! I should I update it to our basic one.

Reply:
I notice that one (not sure static-client or server) takes like 5m to shutdown on pod delete so it might take a long time for your test cleanup.

Reply:
Oh, I see what you mean. It still takes a while to terminate, but because it's a deployment, kubectl doesn't wait for it to finish terminating.